# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1339182683 -7200
#      Fri Jun 08 21:11:23 2012 +0200
# Branch gui
# Node ID c6135951bc1867106fd6f5c8300346a40e7a3905
# Parent  4812bcd524a1c1d5ca53a4e8d86381d7463cc1de
Fixed detecting debugging mode.

* octave-link.cc: Monitoring Vdebugging instead of tree_evaluator::debug_mode.

diff --git a/gui/src/octave-adapter/octave-link.cc b/gui/src/octave-adapter/octave-link.cc
--- a/gui/src/octave-adapter/octave-link.cc
+++ b/gui/src/octave-adapter/octave-link.cc
@@ -67,19 +67,19 @@ octave_link::generate_events ()
   if (current_working_directory != _last_working_directory)
     {
       _last_working_directory = current_working_directory;
       if (_octave_event_listener)
         _octave_event_listener
             ->current_directory_has_changed (_last_working_directory);
     }
 
-  if (_debugging_mode_active != tree_evaluator::debug_mode)
+  if (_debugging_mode_active != Vdebugging)
     {
-      _debugging_mode_active = tree_evaluator::debug_mode;
+      _debugging_mode_active = Vdebugging;
       if (_octave_event_listener)
         {
           if (_debugging_mode_active)
             _octave_event_listener->entered_debug_mode ();
           else
             _octave_event_listener->quit_debug_mode ();
         }
     }
